data_diff/dbt.py

```diff
@@ -3,14 +3,21 @@
 import os
 import time
 import rich
-import yaml
 from dataclasses import dataclass
 from packaging.version import parse as parse_version
 from typing import List, Optional, Dict
 
 import requests
-from dbt_artifacts_parser.parser import parse_run_results, parse_manifest
-from dbt.config.renderer import ProfileRenderer
+
+def import_dbt():
+    try:
+        from dbt_artifacts_parser.parser import parse_run_results, parse_manifest
+        from dbt.config.renderer import ProfileRenderer
+        import yaml
+    except ImportError:
+        raise RuntimeError("Could not import 'dbt' package. You can install it using 'pip install data-diff[dbt]'.")
```

Comment:
Does this need to be `pip install 'data-diff[dbt]'`?

Reply:
I disagree - we should address this particular case in this particular issue. No reason to create an error that is not useful.

We can address the other instances in follow up issues but we should address this one in this issue.

Reply:
So what do you suggest? There is no single form of writing it that will work for everyone.

Reply:
Does `pip install 'data-diff[dbt]'` work for some people and `pip install 'data-diff[dbt]'` work for others?

Reply:
I don't understand the pushback. I know we have an issue though. If I run `pip install data-diff[dbt]` - I get the following "no matches found: data-diff[dbt]". Which may leave users confused and without a next step.

Reply:
Done.